### ISSUE TITLE
PHP error encountered

### ISSUE THREAD

**Issue by hitchy** [bakerframework/bakercloud-ce-console#23]
When installing CloudConsole I got the following error:

A PHP Error was encountered
Severity: Warning
Message: Creating default object from empty value
Filename: install/index.php
Line number: 43

I still get the 'Congratulations, you have successfully installed...' message and can log in. Having checked the database I've created it's populated with tables too.

If I go to my api and hit 'Check DB connectivity' I get the following message:

Fatal error: Class 'PDO' not found in /home/myname/public_html/api/index.php on line 53

Looking at line 43 in the index.php file (the file I'm looking at was located bakercloud-ce-api-master/index/php before uploading via FTP) it's the entry for my database username. 

I've checked this a number of times, including copying and pasting directly from my cPanel MySQL Users listing and it's the correct information. I've check for rogue spaces too and nothing.

As far as I can tell the information is correct and I've followed the screencast exactly. Any ideas what I'm doing wrong and a way to fix the error?

My hosting is a VPS provided by KnownHost (in case that's important), and also, I'm very new to this and little to no coding experience so please keep any responses in layman's terms :)

**Comment by nin9creative** [comment]
Submit a support ticket to make sure that PDO for MySQL is enabled...

Regards,
Andrew

**Comment by hitchy** [comment]
Thanks Andrew, all sorted now.
